src/components/SidebarBottom.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { action } from "@storybook/addon-actions";
```

Comment:
Why not use the new `@storybook/test `here? Isn't that perferred now?

Reply:
I'm not sure actually, I've seen `fn` not result in a log entry in the actions panel, but maybe that was an exception.